src/styles/navbar.css

```diff
@@ -0,0 +1,31 @@
+/*Gestion du style du navbar*/
+.app__navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 2rem 4rem;
+  align-items: center;
+  background-color: var(--dark-blue);
+  font-size: var(--home-font-size);
+  font-family: var(--font-family);
+  border-bottom: 1px solid var(--very-dark-blue-dark-mode);
+}
+.app__navbar-title {
+  font-size: 20px;
+  font-weight: bold;
+}
+.app__navbar-mode {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  cursor: pointer;
+}
+/*Gestion du responsive*/
+@media screen and (max-width: 450px) {
+  .app__navbar {
+    padding: 1rem 1rem;
```

Comment:
Salut @jakemelvin consistency in your css units, I see you use rem's and some other places px. A good ideas will be to define a mixin that converts your px's to rem's

```
@function rem($pixels, $base-font-size: 16px) {
  @return ($pixels / $base-font-size) * 1rem;
}
```

Reply:
So I either have to use pixel or rem is that it? Even in the max-width of my media query?

Reply:
I made the differents changes to the unit used, to have the same units everywhere I also change the units in the different font sizes to use em